src/Radical.sln

```diff
@@ -101,6 +103,22 @@ Global
 		{602FFE39-19E6-4E1B-A496-CFB6D01F2B7A}.Release|ARM.ActiveCfg = Release|Any CPU
 		{602FFE39-19E6-4E1B-A496-CFB6D01F2B7A}.Release|x64.ActiveCfg = Release|Any CPU
 		{602FFE39-19E6-4E1B-A496-CFB6D01F2B7A}.Release|x86.ActiveCfg = Release|Any CPU
+		{31CC206D-2014-4612-811E-82B198385343}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{31CC206D-2014-4612-811E-82B198385343}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{31CC206D-2014-4612-811E-82B198385343}.Debug|ARM.ActiveCfg = Debug|Any CPU
+		{31CC206D-2014-4612-811E-82B198385343}.Debug|ARM.Build.0 = Debug|Any CPU
+		{31CC206D-2014-4612-811E-82B198385343}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{31CC206D-2014-4612-811E-82B198385343}.Debug|x64.Build.0 = Debug|Any CPU
+		{31CC206D-2014-4612-811E-82B198385343}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{31CC206D-2014-4612-811E-82B198385343}.Debug|x86.Build.0 = Debug|Any CPU
+		{31CC206D-2014-4612-811E-82B198385343}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{31CC206D-2014-4612-811E-82B198385343}.Release|Any CPU.Build.0 = Release|Any CPU
+		{31CC206D-2014-4612-811E-82B198385343}.Release|ARM.ActiveCfg = Release|Any CPU
+		{31CC206D-2014-4612-811E-82B198385343}.Release|ARM.Build.0 = Release|Any CPU
+		{31CC206D-2014-4612-811E-82B198385343}.Release|x64.ActiveCfg = Release|Any CPU
+		{31CC206D-2014-4612-811E-82B198385343}.Release|x64.Build.0 = Release|Any CPU
+		{31CC206D-2014-4612-811E-82B198385343}.Release|x86.ActiveCfg = Release|Any CPU
+		{31CC206D-2014-4612-811E-82B198385343}.Release|x86.Build.0 = Release|Any CPU
```

Comment:
Why so many build configuration? Can you please reduce them to the minimum required? E.g. remove the ARM stuff.